### PR TITLE
Replace polished lib

### PR DESCRIPTION
**Pull request by jaketrent**
Here are the preliminary bundle size results:
https://pluralsight.slack.com/archives/C5CMEKH5Z/p1547505040032800

Please have a look at the approach here.  If you dig it, I can apply the change across these other packages that use polished as well:

```
packages/card/package.json
29:    "polished": "^1.9.2",

packages/circularprogress/package.json
26:    "polished": "^1.9.2",

packages/datepicker/package.json
30:    "polished": "^1.9.2",

packages/dialog/package.json
27:    "polished": "^1.9.2",

packages/drawer/package.json
29:    "polished": "^1.9.2",

packages/dropdown/package.json
30:    "polished": "^1.9.2",

packages/form/package.json
31:    "polished": "^1.9.2",

packages/linearprogress/package.json
28:    "polished": "^2.3.1",

packages/row/package.json
28:    "polished": "^1.9.2",

packages/site/package.json
22:    "polished": "^1.9.2"

packages/switch/package.json
27:    "polished": "^1.9.2",

packages/text/package.json
26:    "polished": "^1.9.2",

packages/textarea/package.json
29:    "polished": "^1.9.2",

packages/textinput/package.json
29:    "polished": "^1.9.2",

packages/tooltip/package.json
25:    "polished": "^1.9.2",
```

### CODE REVIEW

File: packages/button/src/css/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import {
   defaultName as themeDefaultName,
   names as themeNames
 } from '@pluralsight/ps-design-system-theme/vars'
-import { transparentize } from 'polished'
+import { transparentize } from '@pluralsight/ps-design-system-util/color'
```
Comment (danethurber) [inline packages/button/src/css/index.js]: what type of things do you think should go in `util`? vs a separate package?

Comment (jaketrent) [reply packages/button/src/css/index.js]: Things that are util, small, insignificant enough to not warrant the ceremony of a specific package...  That's a rather loose definition.

File: packages/dialog/src/react/index.js [hunk 2]
```diff
@@ -110,7 +113,7 @@ class Dialog extends React.Component {
     return (
       <FocusManager
         autofocus={!props.disableFocusOnMount}
-        trapped={!!props.model || !!props.onClose}
+        trapped={!!props.modal || !!props.onClose}
```
Comment (danethurber) [inline packages/dialog/src/react/index.js]: haha - i can never spell that word

**Comment by jaketrent** [comment]
@danethurber This should be ready now.  All polished packages are cleansed.  Please give it another look.

**Pull request merged**